utilities/sourceEvents.py

```diff
@@ -455,7 +455,7 @@ def store(documents):
                 # for image accessing here, we first attempt to download image and if there is indeed an
                 # image in existence. We, then, try to upload the image.
                 imageId = None
-                if downloadImage(result['calendarId'], result['dataSourceEventId'], result['eventId']):
+                if downloadImage(result['originatingCalendarId'], result['dataSourceEventId'], result['eventId']):
```

Comment:
In source_utilities.py, line number 262, the corresponding method call needs to be updated to use originatingCalendarId. Thanks.